res/css/_common.pcss

```diff
@@ -694,6 +694,13 @@ legend {
     color: $username-variant8-color;
 }
 
+.mx_Icon--thread {
```

Comment:
Not mandatory since the current icon stuff is temporary until Compound arrives™

You could add `mx_Icon_18` and `mx_Icon_secondary-content`  [here](https://github.com/matrix-org/matrix-react-sdk/blob/develop/res/css/compound/_Icon.pcss).